commonutils/publish.gradle

```diff
@@ -44,6 +44,13 @@ publish.dependsOn 'assembleRelease'
 
 publishing {
 
+    // use `./gradlew publish` to deploy the artifact locally for testing
+    repositories {
+        maven {
+            url "../repo"
```

Comment:
Where do we define publishing into this directory, and why not just use `mavenLocal()` instead?